### ISSUE TITLE
String index out of range: 0

### ISSUE THREAD

**Issue by ertugrulcetin** [cursive-ide/cursive#1572]
Getting **String index out of range: 0** all the time and can not open my clojure file.

![screen shot 2016-11-07 at 16 58 35](https://cloud.githubusercontent.com/assets/8271291/20064592/9045bd6a-a50b-11e6-8c33-e9957fb35d75.png)

**Comment by cursive-ide** [comment]
Is this for a particular file? What is that file called (name and complete path)? What is the namespace it contains called?

**Comment by ertugrulcetin** [comment]
@cursive-ide 
Sorry just noticed that the ns had two dots `(ns jof-of-clojure.ch8.c-8-1..code-as-data-ex)`

Is it normal to cause a problem like String index exception?

**Comment by cursive-ide** [comment]
No, it shouldn't cause a problem - I'll fix that. But fixing that issue in the ns name should fix the problem for you. I'll leave this open so I don't forget to fix the issue.
